source/components/atoms/Select/Select.tsx

```diff
@@ -42,6 +42,7 @@ interface Props {
   editable?: boolean;
   style?: ViewStyle;
   onBlur: (value: string) => void;
+  onOpen: (event: undefined, isSelect: true) => void;
```

Comment:
The reason `event` is set to `undefined` is due to the `onOpen` event from [react-native-picker-select](https://github.com/lawnstarter/react-native-picker-select#readme) not giving any event details, but the event handler coming from higher up expects positional arguments.

As for the `isSelect`, it can only be `true` since it's coming from the Select component.


It might be better to have a seperate event handler for `Select` and `TextInput` fields.

Reply:
Ah I see! I missed how specific this was to `Select`.